configuration/attributetypes/attribute_types.csv

```diff
@@ -4,4 +4,5 @@ Uuid,Void/Retire,Entity name,Name,Description,Min occurs,Max occurs,Datatype cla
 50e338d4-fae2-4c99-b57e-475e8f23321d,,Program,Primary Care Service,,0,1,org.bahmni.module.bahmnicore.customdatatype.datatype.CodedConceptDatatype,5f16a893-d300-4634-9c45-9485be023f98,,,
 fdd3754d-a069-4e54-8938-22b6492cb2b6,,Program,Priority 1,,0,1,org.bahmni.module.bahmnicore.customdatatype.datatype.CodedConceptDatatype,d47ce963-77fa-494a-ab37-7f0806d41ecb,,,
 6228d457-b9ec-46eb-a15c-9ca07c357352,,Program,Priority 2,,0,1,org.bahmni.module.bahmnicore.customdatatype.datatype.CodedConceptDatatype,c69ff88e-309b-41a0-99ae-5eb58dfc5966,,,
-fd5a4ad0-b527-49ce-b001-3bd6e565783c,,Program,Priority 3,,0,1,org.bahmni.module.bahmnicore.customdatatype.datatype.CodedConceptDatatype,8a6d5b95-ac55-41c2-b27e-ce18012bf962,,,
\ No newline at end of file
+fd5a4ad0-b527-49ce-b001-3bd6e565783c,,Program,Priority 3,,0,1,org.bahmni.module.bahmnicore.customdatatype.datatype.CodedConceptDatatype,8a6d5b95-ac55-41c2-b27e-ce18012bf962,,,
+f245bc0a-6af8-4600-a339-c528d2ce1347,,Program,Status,,0,1,org.bahmni.module.bahmnicore.customdatatype.datatype.CodedConceptDatatype,e6fce7b7-cff5-4506-a909-a7061b6b9840,,,
```

Comment:
Let us change the name to 'Needs Surgery Status' instead of 'Status' to avoid conflict with this [PR](https://github.com/globalhealthcoalition/openmrs-config-ghc/pull/60/files)

Reply:
Keep the Short name as 'Status' and check how the display will be.